src/resources/Incorporation/BC.ts

```diff
@@ -16,7 +16,8 @@ export const IncorporationResourceBc: IncorporationResourceIF = {
     blurb: `Add the people and Corporations/firms who will have a role in your company. People
       can have multiple roles; Corporations/firms can only be Incorporators.`,
     helpSection: null,
-    addOrganization: 'Add a Corporation or Firm',
+    addIncorporator: true,
```

Comment:
I restored this resource property (from before my changes). Ditto for the other resource files below.